apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -160,10 +160,9 @@ const Portfolio = ({ balances, isLoadingAllBalances }: PortfolioProps) => {
                       <StyledNetworkLogoContainer>
                         <TokenIcon
                           size="14px"
-                          token={{
-                            ...tokenInfo.token,
```

Comment:
would this token not exist? we do use the address and the chain id to search for a logo in case the ghTokenListLogoUrl does not exist

Reply:
In order to render a Network logo, I found out that passing the tokenAddress could render an unexpected icon. 
Just passing an empty token with logoUri and it's chainId was a way to make sure about this result

Reply:
Ah gotcha that is the network icon, mmm I'm thinking we should compononentize this

Reply:
I'll create a ticket for it

Reply:
Created BLY-1425 and moved it to the next cycle